src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/LoginThrottlingFactory.php

```diff
@@ -121,7 +122,13 @@ private function registerRateLimiter(ContainerBuilder $container, string $name,
         if (interface_exists(RateLimiterFactoryInterface::class)) {
             $container->registerAliasForArgument($limiterId, RateLimiterFactoryInterface::class, $name.'.limiter');
             $container->registerAliasForArgument($limiterId, RateLimiterFactoryInterface::class, $name);
-            $factoryAlias->setDeprecated('symfony/security-bundle', '7.4', 'The "%alias_id%" autowiring alias is deprecated and will be removed in 8.0, use "RateLimiterFactoryInterface" instead.');
+            $factoryAlias->setDeprecated('symfony/security-bundle', '7.4', \sprintf('The "%%alias_id%%" autowiring alias is deprecated and will be removed in 8.0, use "%s $%s" instead.', RateLimiterFactoryInterface::class, (new Target($name.'.limiter'))->getParsedName()));
+
+            $internalAliasId = \sprintf('.%s $%s.limiter', RateLimiterFactory::class, $name);
+
+            if ($container->hasAlias($internalAliasId)) {
```

Comment:
`registerAliasForArgument()` can register a second alias (that's what I call "internal alias" here): https://github.com/symfony/symfony/blob/ae30d7c04d249fd07758582d67fc5ea014363f9a/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L1475-L1477